packages/backend/src/scheduler/SchedulerTaskTracer.ts

```diff
@@ -0,0 +1,255 @@
+import {
+    SCHEDULER_TASKS,
+    SchedulerTaskName,
+    TaskPayloadMap,
+} from '@lightdash/common';
+import * as Sentry from '@sentry/node';
+import { JobHelpers, Task, TaskList } from 'graphile-worker';
+import moment from 'moment';
+import ExecutionContext from 'node-execution-context';
+import { ExecutionContextInfo } from '../logging/winston';
+import { TypedTask, TypedTaskList } from './types';
+
+const getTagsForTask: {
```

Comment:
in the future we can just add more tags here with type-safety now that payload is task-aware.